org.eclipse.scout.rt.server.commons/src/main/java/org/eclipse/scout/rt/server/commons/servlet/filter/gzip/GzipServletOutputStream.java

```diff
@@ -22,6 +22,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * @deprecated will be removed in a future release.
+ */
+@Deprecated
 public class GzipServletOutputStream extends ServletOutputStream {
 
   private static final Logger LOG = LoggerFactory.getLogger(GzipServletOutputStream.class);
```

Comment:
Line 137: could that be a feature we will miss in the new implementation? Could you check if such a try / catch would be possible using the new handler?

Reply:
`org.eclipse.jetty.server.handler.gzip.GzipRequest` and `org.eclipse.jetty.server.handler.gzip.GzipResponseAndCallback` seem not to use traditional (blocking) streams (maybe this is the difference to a traditional filter).

However I will try to reproduce it on our test environment upload/download some large files, check whether download was gzipped and simulate a connection error (if file is large enough I may just stop reading) - afterwards check if exceptions are logged.